### PR TITLE
Responsive Menu Button

### PR DESCRIPTION
**Pull request by David-Lanzz**
- We added a responsive menu to display when the hamburger is clicked in the mobile version
- We linked the menu items to their corresponding cards in the HTML page
- We added transitions to the menu button for visual entertainment to the user when clicked

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -46,11 +46,47 @@ nav {
...
+  line-height: 2rem;
+}
+
+.menuList a {
+  font-weight: 600;
+  font-size: 25px;
+  line-height: 44px;
+  color: #fff;
+}
+
```
Comment (IngDavidTM) [inline styles.css]: - [ ] Good job styling your project :100: , but as a project requirement you should stick as much as possible to the Figma template. As you can see below the font size of your elements and the space between them should be bigger :+1: and also the background should be blur.

**Your design**

Screenshot from 2023-01-30 08-44-56

**Figma**

Screenshot from 2023-01-30 08-44-32

File: script.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const hamburger = document.querySelector('#menu');
+const disappear = document.querySelector('#disappear');
+const rotateRight = document.querySelector('#rotate-right');
+const rotateLeft = document.querySelector('#rotate-left');
+const menuList = document.querySelector('#menuList');
+const menuButtons = document.querySelectorAll('.menuButton');
```
Comment (IngDavidTM) [inline script.js]: - [ ] Kindly don't forget that it is required when the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear, to improve the user experience :+1:

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @David-Lanzz  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

- [ ] While you did a fantastic job implementing the functionality on your project :1st_place_medal: , as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) your project should have commits from you and your parent to make sure that everybody has contributed code on each project. If you don't have a partner, please let the next reviewer know :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**